tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@tsconfig/node16",
+  "extends": "./node_modules/@tsconfig/node16/tsconfig.json",
```

Comment:
Can you version this to node18 given the updates to the default node version

Reply:
Done.